### PR TITLE
Contributor roles

### PR DESCRIPTION
**Pull request by Scrumplex**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        repo: 'Translations',
+
+        key: 'translations',
+        name: 'Translator',
+      },
+    ],
+    cacheSec: 3600,
+    updateJobCron: '0 */10 * * * *', // every 10 minutes
+  },
+  expressPort: Number(process.env.EXPRESS_PORT) || 3000,
```
[the remark this answers is not in the document]
Comment (TheKodeToad) [reply src/config.ts:42]: :train:

File: src/server.ts [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+const getGitHubConnections = async (rest: REST) => {
+  const connections = (await rest.get(
+    Routes.userConnections()
+  )) as RESTGetAPICurrentUserConnectionsResult;
+
+  return connections.reduce((result, connection) => {
+    if (connection.type == ConnectionService.GitHub) result.push(connection.id);
+    return result;
+  }, [] as string[]);
```
Comment (TheKodeToad) [inline src/server.ts:70]:
```suggestion
  return connections
    .filter(connection => connection.type === ConnectionService.GitHub)
    .map(connection => connection.id);
```

**Review by TheKodeToad — APPROVED** [review]
Toad's official seal of approval
(I do not have much knowledge in this area)
Just one suggestion that may improve code quality but it seems you intentionally avoided?

File: src/server.ts (hunk 2 above)
Comment (Scrumplex) [reply src/server.ts:70]: I intentionally avoided this, as it would result in two iterations, instead of just one using reduce

Comment (TheKodeToad) [reply src/server.ts:70]: hm, i can't imagine any big performance impact

**Comment by TheKodeToad** [comment]
Conflicts with main!

**Comment by TheKodeToad** [comment]
Hopefully I didn't mess up the merging... :P

**Pull request closed without merging**